src/components/review/AddNewReviewForm.tsx

```diff
@@ -31,10 +35,17 @@ export const AddNewReviewForm = ({ match, onClose }: Props) => {
 
   const [newReview, setNewReview] = useState(newReviewInitalState);
 
-  const addReview = (event: FormEvent<HTMLFormElement>) => {
+  const dispatch = useDispatch();
+
+  const submitReview = (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     postReview(newReview)
-      .then(() => {
+      .then(({ review, rating }: ReviewRating) => {
+        unstable_batchedUpdates(() => {
```

Comment:
unstable? :P

Reply:
This is fine

> I know that `unstable_batchedUpdates()` is, uh... "unstable". However, the React team has stated that "it's the most stable of the unstable APIs", and "half of Facebook depends on this".

https://github.com/facebook/react/issues/15080#issue-419249211